lines.js

```diff
@@ -8,61 +8,84 @@ var sizePanel = document.querySelector("#dot-size-panel");
 var dotSize;
 var radius = document.querySelector(".dot-size.chosen").dataset.size;
 var mousePressed = false;
+var projectHistory = [];
+var historyBack = document.querySelector(".history-button.backward");
+var historyForward = document.querySelector(".history-button.forward");
+var currentIndex = 0;
+var isOut;
 
 myCanvas.width = document.body.clientWidth;
 
-myCanvas.addEventListener('mousedown', drawPoint, false);
-myCanvas.addEventListener('mousemove', drawLine, false);
+myCanvas.addEventListener('mousedown', startDraw, false);
+document.addEventListener('mousemove', drawLine, false);
 document.addEventListener('mouseup', stopDrawing, false);
+window.addEventListener('keydown', keyPressed, false);
 
 button1.addEventListener('click', buttonState, false);
 
 colourPanel.addEventListener('click', chooseColour, false);
 sizePanel.addEventListener('click', chooseSize, false);
 
+historyBack.addEventListener('click', turnBack, false);
+historyForward.addEventListener('click', turnForward, false);
+
 var myColour = document.querySelector(".colour.chosen").dataset.colour;
 context.strokeStyle = myColour;
 context.fillStyle = myColour;
+context.lineWidth = 2 * radius;
 
 function buttonState(e) {
```

Comment:
1. `buttonState` is strange name for the function. `buttonState` what?
I agree with `toggleButtonState`, `checkButtonState` or `getButtonState`.

2. As i can see, you do not use `e`. Useless parameters is not good approach